spec/models/user_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
```

Comment:
The card didn't specify to add tests for user. Beware from now on to comply with the scope of the card you are working on.